### DISCUSSION TITLE
Does medusa support cash on delivery?

### DISCUSSION THREAD

**Discussion by tsvetann** [medusajs/medusa#3843] [Q&A]
Hi all,

I am curious if medusa supports cash on delivery. A customer orders something and pays for his order only when the items are delivered to him

Thanks

**Comment by Labham-Jain** [comment]
I think setting payment provider to "manual" can be used as cash on delivery...

**Comment by tsvetann** [comment]
I am currently using manual payment provider as you suggested but it feels a bit like a work around, especially that the Admin requires a manual payment capturing action, but the payment is actually collected by the deliver service

**Comment by ghost** [comment]
@tsvetann Did you find any working solution?

**Reply by tsvetann** [reply]
Manual payment

**Reply by SalahAdDin** [reply]
In which way?
